templates/coderd.yaml

```
@@ -176,6 +176,12 @@ spec:
             - name: SSL_SERVER_KEY_FILE
               value: {{ join ":" $serverKeys }}
             {{- end }}
+            {{- if .Values.coderd.oidc.enableRefresh }}
+            - name: ENABLE_OIDC_REFRESH
```

```suggestion
            - name: OIDC_ENABLE_REFRESH
```
so they both start with OIDC

---

true